pkg/core/consensus/user/README.md

```diff
@@ -1,10 +1,54 @@
 # Provisioners and Sortition
+This package implements the data structure which holds the _Provisioners_ set, and implements methods to extract _Block Generators_ and _Voting Committees_, which are responsible to propose and decide on new blocks during the SA consensus protocol.
 
-This package implements the data structure which holds the Provisioner committee, and implements methods on top of this committee in order to be able to extract **voting committees** which are eligible to decide on blocks during the SBA\* consensus protocol.
+The extraction procedure is implemented by the `CreateVotingCommittee` function, which takes a committee `size` as input and outputs a _Voting Committee_ with an equal number of members.
```

Comment:
`with an equal number of members`

This is not always true.
The voting committee has a number of member equal or lower than the desired `size`
The total cluster weight is, indeed, equal to the desired `size`